run-cnf-suites.sh

```diff
@@ -162,4 +162,7 @@ cd ./cnf-certification-test || exit 1
 	${FOCUS_STRING} \
 	${SKIP_STRING} \
 	"${LABEL_STRING}" \
-	${GINKGO_ARGS}
+	${GINKGO_ARGS} |& tee $OUTPUT_LOC/tnf-execution.log
+
+# preserving the exit status
+exit ${PIPESTATUS[0]}
```

Comment:
Add the new line at the end. I think your editor doesn't add the new line automatically. What do you use? As far as I know, all *nix text editors do it.

Reply:
fixed